src/Backoffice/Employee/Domain/Models/Employee.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employee\Domain\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+use Illuminate\Notifications\Notifiable;
+use Lightit\Backoffice\Task\Domain\Models\Task;
+use Lightit\Shared\App\Timestampable;
+
+/**
+ * @property int                        $id
+ * @property string                     $name
+ * @property string                     $email
+ * @property \Illuminate\Support\Carbon $created_at
+ * @property \Illuminate\Support\Carbon $updated_at
+ * @property-read \Illuminate\Database\Eloquent\Collection<int, Task> $tasks
+ *
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee newModelQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee newQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee query()
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee whereCreatedAt($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee whereEmail($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee whereId($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee whereName($value)
+ * @method static \Illuminate\Database\Eloquent\Builder<static>|Employee whereUpdatedAt($value)
+ *
+ * @property-read \Illuminate\Notifications\DatabaseNotificationCollection<int, \Illuminate\Notifications\DatabaseNotification> $notifications
+ * @property-read int|null $notifications_count
+ *
+ * @mixin \Eloquent
+ */
+class Employee extends Model implements Timestampable
+{
+    use Notifiable;
+
+    protected $fillable = [
+        'name',
+        'email',
+    ];
```

Comment:
Do the opposite, declare a $guarded and specify which ones you do not want to be mass assignable

```suggestion
    protected $guarded = [
        'id',
    ];
```